### DISCUSSION TITLE
Question: how to use CH with a complete image

### DISCUSSION THREAD

**Discussion by selroc** [cloud-hypervisor/cloud-hypervisor#3452] [Q&A] [answered]
When the kernel is included in the image, how do we launch the VM ?
I guess we will not specify a --kernel option.

**Comment by rbradford** [comment, marked as answer]
Download https://github.com/cloud-hypervisor/rust-hypervisor-firmware/releases/download/0.3.2/hypervisor-fw or https://github.com/cloud-hypervisor/edk2/releases/download/ch-40fbc7e24c/OVMF.fd and use as the `--kernel`

**Reply by selroc** [reply]
Thank you.
